versions/pre/1.17/1.17-rc1/joined.tsrg

```diff
@@ -134623,7 +134623,7 @@ enf net/minecraft/src/C_4134_ 4134
 		2 o p_109712_ 109712
 	a (IIIZ)V m_109501_ 109501
 		0 o p_109502_ 109502
-		1 o p_109503_ 109503f
+		1 o p_109503_ 109503
```

Comment:
As I ping on discord, things like this should be added to the verifier so that in the future they are detected and fixed.
Typos happen, but it's best to ruggedize against them for the future.